src/lib.rs

```diff
@@ -21,12 +21,8 @@ pub(crate) fn app_dir(app_file: impl AsRef<Path>) -> Result<PathBuf> {
     Ok(path_buf)
 }
 
-pub(crate) fn write_failed_msg(bindle_id: &bindle::Id, dest_dir: &Path) -> String {
-    format!(
-        "Failed to write bindle '{}' to {}",
-        bindle_id,
-        dest_dir.display()
-    )
+pub(crate) fn write_failed_msg(dest_dir: &Path) -> String {
+    format!("Failed to write bindle to {}", dest_dir.display())
```

Comment:
It might be good to keep the bindle ID for diagnostic purposes, though this was probably more relevant to prepare-type scenarios than to the current one-stop deploy mode.

Reply:
Also fixed!